tests/integration/targets/ndo_synce_interface_policy/tasks/main.yml

```diff
@@ -0,0 +1,266 @@
+
+    - name: Create a fabric template
+      cisco.mso.ndo_template:
+        <<: *mso_info
+        name: ansible_fabric_policy_template
+        type: fabric_policy
+        state: present
+
+    # CREATE
+    - name: Create a new syncE interface policy (check mode)
+      cisco.mso.ndo_synce_interface_policy: &add_synce_interface_policy
+        <<: *mso_info
+        template: ansible_fabric_policy_template
+        interface_policy: ansible_test_synce_interface_policy
+        state: present
+      check_mode: true
+      register: cm_add_synce_interface_policy
+
+    - name: Create a new syncE interface policy
+      cisco.mso.ndo_synce_interface_policy:
+        <<: *add_synce_interface_policy
+      register: nm_add_synce_interface_policy
+
+    - name: Create a syncE interface policy again
+      cisco.mso.ndo_synce_interface_policy:
+        <<: *add_synce_interface_policy
+      register: nm_add_synce_interface_policy_again
+
+    - name: Assert syncE interface policy was created
+      assert:
+        that:
+          - cm_add_synce_interface_policy is changed
+          - cm_add_synce_interface_policy.previous == {}
+          - cm_add_synce_interface_policy.current.name == "ansible_test_synce_interface_policy"
+          - cm_add_synce_interface_policy.current.srcPriority == 100
+          - cm_add_synce_interface_policy.current.waitToRestore == 5
+          - nm_add_synce_interface_policy is changed
+          - nm_add_synce_interface_policy.previous == {}
+          - nm_add_synce_interface_policy.current.name == "ansible_test_synce_interface_policy"
+          - nm_add_synce_interface_policy.current.srcPriority == 100
+          - nm_add_synce_interface_policy.current.waitToRestore == 5
+          - nm_add_synce_interface_policy_again is not changed
+          - nm_add_synce_interface_policy_again.previous.name == nm_add_synce_interface_policy_again.current.name == "ansible_test_synce_interface_policy"
+          - nm_add_synce_interface_policy_again.previous.srcPriority == nm_add_synce_interface_policy_again.current.srcPriority == 100
+          - nm_add_synce_interface_policy_again.previous.waitToRestore == nm_add_synce_interface_policy_again.current.waitToRestore == 5
+          - nm_add_synce_interface_policy_again.previous.description == nm_add_synce_interface_policy_again.current.description == ""
+          - nm_add_synce_interface_policy_again.previous.uuid is defined
+          - nm_add_synce_interface_policy_again.current.uuid is defined
+
+    # UPDATE
+
+    - name: Update a syncE interface policy (check mode)
+      cisco.mso.ndo_synce_interface_policy: &update_synce_interface_policy
+        <<: *add_synce_interface_policy
+        description: changed_description
+        admin_state: enabled
+        sync_state_msg: enabled
+        selection_input: enabled
+        src_priority: 110
+        wait_to_restore: 12
+        state: present
+      check_mode: true
+      register: cm_update_synce_interface_policy
```

Comment:
Can you add the proposed value check for the check mode tests?

Reply:
Proposed value check will not be added now, since the existing modules with proposed gives the below error 
----> 'dict object' has no attribute 'proposed'

This will be fixed later.